ui/packages/security-servlet-logout/src/main/resources/logout-response.html

```diff
@@ -27,9 +27,8 @@
 <body class="full-width logout-response-body">
 <div class="full-width logout-response-container">
     <div class="logout-msg">
-        <h2>You are now signed out.</h2>
+        <h2 id="extramessage"></h2>
```

Comment:
I'm surprised that this whole time we've been saying "You are now signed out." in all the various failure scenarios. Good catch 👍